source/shared/cpp/ObjectModel/MarkDownParsedResult.cpp

```diff
@@ -0,0 +1,214 @@
+#include "MarkDownParsedResult.h"
+
+using namespace AdaptiveCards;
+
+void MarkDownParsedResult::Translate()
+{
+    MatchLeftAndRightEmphasises();
+}
+
+// appends html block tags at head and tail of the list
+void MarkDownParsedResult::AddBlockTags()
+{
+    // if html list tag, add <ul>, otherwise <p> at the head of the list
```

Comment:
>// if html list tag, add <ul>, otherwise <p> at the head of the list [](start = 4, length = 68)

[nit] Update this comment since we're not doing the tags here anymore